administrator-guides/custom-sounds/README.md

```diff
@@ -0,0 +1,27 @@
+# Custom Sounds
+
+Rocket.Chat allows you to add cutom sound that can be used as notification sound by other users for different channels.
```

Comment:
Maybe `custom sound files that can be used as the notification's sounds. Users can customize their notification sound preferences for specific channels`